point_cloud_test/src/queries.rs

```diff
@@ -0,0 +1,39 @@
+// Some synthetic queries for synthetic data. These are just examples, more can be added.
+use crate::synthetic_data::SyntheticData;
+use crate::S2_LEVEL;
+use cgmath::{PerspectiveFov, Rad, Vector3};
+use point_viewer::iterator::PointLocation;
+use point_viewer::math::{Frustum, Obb, S2Point};
+
+pub fn get_abb_query(data: SyntheticData) -> PointLocation {
+    let abb = data.bbox();
+    PointLocation::Aabb(abb)
+}
+
+pub fn get_obb_query(data: SyntheticData) -> PointLocation {
+    let obb = Obb::new(
+        data.ecef_from_local().clone(),
+        Vector3::new(15.0, 15.0, 15.0),
```

Comment:
nit: also here, would it be useful to move these number into costants?

Reply:
Made them data-dependent.